content/kotlin/concepts/datetime/terms/toDateTimePeriod/toDateTimePeriod.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.toDateTimePeriod()'
+Description: 'Interprets an ISO-8601 formatted duration string, converting it into a DateTimePeriod object. In cases where the string's time components are either missing or amount to zero, the function yields a DatePeriod instead.'
```

Comment:
Make this short and descriptive. This long explanation is supposed to be done after the meta data not here.

Reply:
Noted with thanks!